atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/TodoEteTest.java

```diff
@@ -15,8 +15,8 @@
  */
 package com.palantir.atlasdb.ete;
 
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.hasItem;
+import org.hamcrest.MatcherAssert;
```

Comment:
Again, we should statically import from the proper classes (I'm surprised it isn't Matchers for `hasItem`..)